lib/commands/folder.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * Voicemail Administrator Folder Commands.
+ *
+ * @module voicemail-admin
+ * @copyright 2015, Digium, Inc.
+ * @license Apache License, Version 2.0
+ * @author Jonathan R. Rose <jrose@digium.com>
+ */
+
+'use strict';
+
+var dal;
+var logger;
+var sprintf = require('sprintf').sprintf;
+
+/**
+ * Removes keys from an associative array that are either functions or objects
+ *
+ * @param {object} associativeArray - associative array to cull junk from
+ */
+function removeNonValues(associativeArray) {
```

Comment:
Instead of defining this twice in the mailbox and folder commands, extract it into lib/commands/common.js or lib/commands/util.js.